### PR TITLE
Axe accessibilty

### PR DESCRIPTION
**Pull request by demihennelly**

### CODE REVIEW

File: src/main/java/uk/gov/dvsa/mot/fixtures/generic/WebStepDefinitions.java [hunk 1]
```diff
@@ -134,6 +134,9 @@ public WebStepDefinitions(WebDriverWrapper driverWrapper) {
         Then("^The page title contains \"([^\"]+)\"$", (String expected) ->
                 driverWrapper.checkCurrentPageTitle(expected));
 
+        Then("^I create an accessibility report \"([^\"]*)\"$", (String filename) ->
```
Comment (karlbeckettbjss) [inline src/main/java/uk/gov/dvsa/mot/fixtures/generic/WebStepDefinitions.java:140]: We need to add this step to mot-automated-testsuite/docs/step-definitions.md

File: src/main/resources/features/accessibility/01 - tester.feature [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  Scenario: Accessibility report for login page
+    When I browse to /
+    Then The page title contains "Sign in"
+    And I create an accessibility report "login"
+
+  Scenario: Accessibility report for test homepage and your profile
+    When I login with 2FA using "MOT_TESTER_CLASS_4" as {username1}, {site}
+    Then The page title contains "Your home"
+    And I create an accessibility report "tester_homepage"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/accessibility/01 - tester.feature:12]: Could we use the page name for the report name?

File: src/main/resources/features/accessibility/02 - moth.feature [hunk 3]
```diff
@@ -0,0 +1,44 @@
+@mothaccessibility
+Feature: MOT History accessibility reports
```
Comment (karlbeckettbjss) [inline src/main/resources/features/accessibility/02 - moth.feature:2]: should we add recalls and certificate downlands into this journey?
It depends if these new accessibility report steps are going to be added into the current tests/

**Review by karlbeckettbjss — CHANGES_REQUESTED** [review]
We might need to rebase because of the recent changes..

Karl or Ibrahim will make these changes

**Comment by ibrahimjassat** [comment]
> We might need to rebase because of the recent changes..
> 
> Karl or Ibrahim will make these changes

rebased and pushed back up

**Review by karlbeckettbjss — APPROVED** [review]

**Review by ibrahimjassat — APPROVED** [review]

**Pull request merged**